common/common.cpp

```diff
@@ -192,12 +192,46 @@ bool gpt_params_parse(int argc, char ** argv, gpt_params & params) {
                 break;
             }
             params.rope_freq_scale = std::stof(argv[i]);
+        } else if (arg == "--rope-scaling") {
+            if (++i >= argc) {
+                invalid_param = true;
+                break;
+            }
+            std::string value(argv[i]);
+            /**/ if (value == "none")   { params.rope_scaling_type = LLAMA_ROPE_SCALING_NONE; }
```

Comment:
does "none" make sense? isnt it just "linear" with a factor of 1.0 ?

Reply:
I thought it would make sense to provide a simple way to disable the RoPE scaling if the GGUF has it enabled.

Reply:
alright, sounds reasonable.